packages/modules/src/utils/Types.ts

```diff
@@ -81,27 +104,38 @@ export type CreateSessionDataResponse = {
 };
 
 export interface CreateSessionDataParams {
+  /** window end for the session key */
   validUntil: number;
+  /** window start for the session key */
   validAfter: number;
   sessionValidationModule: Hex;
   sessionPublicKey: Hex;
   sessionKeyData: Hex;
+  /** if multiple session keys are there can pass the uuid here to filter that session key EOA */
```

Comment:
we generate uuid based sessionId. but if you prefer to track it on your side and attach custom session identifier this can be passed